MapleServer2/Packets/SkillBookTreePacket.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Types;
+
+namespace MapleServer2.Packets {
+    public static class SkillBookTreePacket {
+        private static int count = 0;
+        public static Packet Open(Player character) {
+            var pWriter = PacketWriter.Of(SendOp.SKILL_BOOK_TREE);
+
+            // Writes only skills that are learned and for the job rank tab that is opened also doesn't write default passive skills
+            pWriter.WriteByte(0); // Mode (0 = open) (1 = save)
```

Comment:
There's a few other modes (like 2 and 4).
2 is long, unicodestring, byte
4 is int
Just putting this here for more info

Reply:
Can you provide any use cases or sniffs for the other modes? I haven't seen those modes used.